packages/calcite-components/src/components/shell/shell.tsx

```diff
@@ -131,6 +137,24 @@ export class Shell implements ConditionalSlotComponent {
     });
   };
 
+  handlePanelTopChange = (event: Event): void => {
+    const panelTop = event.target as HTMLSlotElement;
```

Comment:
event.target here is the slot element. We don't want to use that, instead we need to know if the slot element has assigned elements.